### PR TITLE
♻️ Wrap creating web socket creation with try/catch and make client operation silent

### PR DESCRIPTION
**Pull request by miracozkan**

### CODE REVIEW

File: lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt [hunk 1]
```diff
@@ -68,7 +67,8 @@ class OkHttpTransport(
         mapOf(
             "topic" to topic,
             "type" to type,
-            "payload" to payload
+            "payload" to payload,
+            "silent" to true
```
Comment (yasin-ce) [inline lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt:71]: What's the purpose of this line? @miracozkan

File: lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt [hunk 2]
```diff
@@ -99,25 +99,27 @@ class OkHttpTransport(
     override fun onFailure(webSocket: WebSocket, t: Throwable, response: Response?) {
         super.onFailure(webSocket, t, response)
         statusHandler(Error(t))
-        disconnected()
+        disconnected(false)
```
Comment (yasin-ce) [inline lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt]: Just by reading this, I can't understand what's false stands for.

File: lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt [hunk 3]
```diff
@@ -99,25 +99,27 @@ class OkHttpTransport(
...
     }
 
     override fun onClosed(webSocket: WebSocket, code: Int, reason: String) {
         super.onClosed(webSocket, code, reason)
-        disconnected()
+        disconnected(true)
     }
 
-    private fun disconnected() {
+    private fun disconnected(forceToClearSession: Boolean) {
```
Comment (yasin-ce) [inline lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt]: `forceToClearSession` sounds like a function name.

**Review by yasin-ce — APPROVED** [review]

File: lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt (hunk 1 above)
Comment (miracozkan) [reply lib/src/main/kotlin/org/walletconnect/impls/OkHttpTransport.kt:71]: It's about sending push notifications. No need to force servers to send notifications based on client actions.

**Pull request merged**